src/main/java/edu/kit/ipd/dbis/database/connection/tables/Table.java

```diff
@@ -223,7 +231,8 @@ protected String getPropertyColumns()
 		for (String element : list) {
 			columns += (element + ", ");
 		}
-		return columns.substring(0, columns.length() - 2);
+		columns = (columns.length() > 0) ? (columns.substring(0, columns.length() - 2)) : (columns);
```

Comment:
that looks pretty cool, because it's so hard to read. Please do not use the ternary operator in such complex statements